Database/Entities/Match.cs

```diff
@@ -117,8 +117,17 @@ public class Match : UpdateableEntityBase, IAuditableEntity<MatchAudit>, IProces
     /// </summary>
     public ICollection<RatingAdjustment> PlayerRatingAdjustments { get; set; } = new List<RatingAdjustment>();
 
+    /// <summary>
+    /// A collection of <see cref="MatchAudit"/>s, one for each change made to this entity
+    /// </summary>
```

Comment:
Technically this isn't required, it inherits the generic documentation from the interface